Dockerfile

```diff
@@ -1,12 +1,27 @@
-FROM adoptopenjdk:11-jre-hotspot
-
-RUN apt-get update \
-    && apt-get install -y unzip \
-    && rm -rf /var/lib/apt/lists/*
+ARG VERSION=3.6
```

Comment:
Defaulting the version means we we would have to update this file every time we release a new version. My current preference is to not have a default for this.

Reply:
Reverted - 4b7c819